### PR TITLE
Feature gui update

### PR DESCRIPTION
**Pull request by joseph-anderson**

### CODE REVIEW

File: core/src/io/github/teamfractal/RoboticonQuest.java [hunk 1]
```diff
@@ -220,31 +207,37 @@ public int getPlayerInt(){
 	public void nextPlayer(){
 		if (this.currentPlayer == playerList.size() - 1){
 			if(isGameEnded()){
-				endGame();
+				setScreen(scoreScreen);
+				phase=7;
```
Comment (JorgenFreeman) [inline core/src/io/github/teamfractal/RoboticonQuest.java]: No longer needed

File: core/src/io/github/teamfractal/actors/RoboticonMarketActors.java [hunk 2]
```diff
@@ -145,102 +144,89 @@ public void changed(ChangeEvent event, Actor actor) {
...
 
-		final TextButton nextButton = new TextButton("Next ->", game.skin);
-		nextButton.addListener(new ChangeListener() {
-			@Override
-			public void changed(ChangeEvent event, Actor actor) {
-				game.nextPhase();
-			}
-		});
+		align(Align.center);
+		//padTop(30);
```
Comment (JorgenFreeman) [inline core/src/io/github/teamfractal/actors/RoboticonMarketActors.java]: Remove commented code

File: core/src/io/github/teamfractal/actors/ScoreScreenActors.java [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+import java.util.ArrayList;
+import java.util.HashMap;
+
+public class ScoreScreenActors extends Table{
+    private RoboticonQuest game;
+    private ScoreScreen screen;
+
+        private Label topText;
+        private Label playerStats;
+        private Label marketStats;
```
Comment (JorgenFreeman) [inline core/src/io/github/teamfractal/actors/ScoreScreenActors.java]: Indentation

File: core/src/io/github/teamfractal/actors/ScoreScreenActors.java [hunk 4]
```diff
@@ -0,0 +1,80 @@
...
+        exitButton.addListener(new ChangeListener() {
+            @Override
+            public void changed(ChangeEvent event, Actor actor) {
+                game.dispose();
+                Gdx.app.exit();
+            }
+        });
+
+        align(Align.center);
+        //padTop(30);
```
Comment (JorgenFreeman) [inline core/src/io/github/teamfractal/actors/ScoreScreenActors.java]: Commented code

File: core/src/io/github/teamfractal/screens/ScoreScreen.java [hunk 5]
```diff
@@ -0,0 +1,93 @@
...
+import io.github.teamfractal.actors.ScoreScreenActors;
+
+public class ScoreScreen extends AbstractAnimationScreen implements Screen {
+
+    final RoboticonQuest game;
+    final Stage stage;
+    final Table table;
+    private ScoreScreenActors actors;
+
+
```
Comment (JorgenFreeman) [inline core/src/io/github/teamfractal/screens/ScoreScreen.java]: Remove extra whitespace

File: core/src/io/github/teamfractal/screens/ScoreScreen.java [hunk 6]
```diff
@@ -0,0 +1,93 @@
+package io.github.teamfractal.screens;
+
+import com.badlogic.gdx.Gdx;
+import com.badlogic.gdx.Screen;
+import com.badlogic.gdx.graphics.GL20;
+import com.badlogic.gdx.scenes.scene2d.Stage;
+import com.badlogic.gdx.scenes.scene2d.ui.Table;
+import com.badlogic.gdx.utils.Align;
+import com.badlogic.gdx.utils.viewport.ScreenViewport;
+
```
Comment (JorgenFreeman) [inline core/src/io/github/teamfractal/screens/ScoreScreen.java]: Remove whitespace

File: core/src/io/github/teamfractal/actors/RoboticonMarketActors.java [hunk 7]
```diff
@@ -145,102 +144,89 @@ public void changed(ChangeEvent event, Actor actor) {
...
+		row().height(20);
+		add().colspan(span);
 		row();
 
-		add();
-		add();
-		add();
-		add();
+		//Row 11
+		//add(nextButton).colspan(span);
```
Comment (JorgenFreeman) [inline core/src/io/github/teamfractal/actors/RoboticonMarketActors.java]: Remove commented code

File: core/src/io/github/teamfractal/actors/HomeMainMenu.java [hunk 8]
```diff
@@ -18,7 +18,7 @@
 	private TextButton btnNewGame;
 	private TextButton btnExit;
 
-	private static Texture titleTexture = new Texture(Gdx.files.internal("roboticon_images/Roboticon_Quest_Title"));
+	private static Texture titleTexture = new Texture(Gdx.files.internal("roboticon_images/roboticon_quest_title.png"));
```
Comment (JorgenFreeman) [inline core/src/io/github/teamfractal/actors/HomeMainMenu.java]: Could you make this filepath a constant

File: core/src/io/github/teamfractal/actors/RoboticonMarketActors.java [hunk 9]
```diff
@@ -51,7 +53,7 @@ public RoboticonMarketActors(final RoboticonQuest game, RoboticonMarketScreen sc
 		widgetUpdate();
 
 		// Buy Roboticon Text: Top Left
-		final Label lblBuyRoboticon = new Label("Purchase Roboticons:", game.skin);
+		final Label lblBuyRoboticon = new Label("PURCHASE ROBOTICONS IN THIS SECTION", game.skin);
```
Comment (JorgenFreeman) [inline core/src/io/github/teamfractal/actors/RoboticonMarketActors.java]: calm down please

File: core/src/io/github/teamfractal/entity/Player.java [hunk 10]
```diff
@@ -275,7 +276,6 @@ public synchronized boolean purchaseLandPlot(LandPlot plot){
 		landList.add(plot);
 		this.setMoney(this.getMoney() - 10);
```
Comment (JorgenFreeman) [inline core/src/io/github/teamfractal/entity/Player.java]: Can you put this 10 into a constant

File: core/src/io/github/teamfractal/actors/RoboticonMarketActors.java (hunk 9 above)
Comment (joseph-anderson) [inline core/src/io/github/teamfractal/actors/RoboticonMarketActors.java]: Caps for clarity as a heading, we could add another skin instead but I'll have to find out how to do that

Comment (JorgenFreeman) [inline core/src/io/github/teamfractal/actors/RoboticonMarketActors.java]: It's fine for now

**Pull request merged**